ansible/site.yml

```diff
@@ -20,131 +13,173 @@
     - name: Local - whoami
       local_action: command whoami
       register: whoami
+
+    - name: Create Local Results Directory
+      local_action:  
```

Comment:
The `local_action` here is unnecessary, since the play it's running in is already `connection: local`.